integration-test/test-data/input.txt

```diff
@@ -0,0 +1 @@
+Contents
```

Comment:
Is this file being used?

Reply:
No, it's not used. Do we want to delete the file?